src/first_steps/intro.md

```diff
@@ -1,78 +1,119 @@
 ## Basic Rizin Usage
 
-The learning curve is usually somewhat steep at the beginning. Although after an hour of using it you should easily understand how most things work, and how to combine the various tools rizin offers.
+The initial learning curve tends to be relatively steep. However, within an hour of usage, you should grasp the workings of most features and learn how to effectively utilize the different tools provided by `rizin`.
 
-To get a first overview over the basic commands just run Rizin and type `?`:
+For a quick introduction to basic commands, execute `rizin` on the command line and enter `?` to obtain an overview.
 
 ```bash
-> ?
-| ![!]                  # Run given commands as in system(3) or shows command history
+[0x00000000]> ?
+| ![!]                  # Run given commands as in system(3) or shows command history (see !?? for more details)
 ...
 | a[?]                  # Analysis commands
 ...
-| p[cdghtm?]            # Print commands
+| p[?]                  # Print commands
 ...
 
 ```
 
-**Note**: you can prepend a `?` after any command to see it's help.
+**Note**: You have the option to add a `?` before any command to access its help.
```

Comment:
maybe we should mention about `??` which shows more info and examples.

Reply:
Definitely